x/ccv/consumer/keeper/keeper.go

```diff
@@ -209,11 +212,11 @@ func (k Keeper) SetProviderChannel(ctx sdk.Context, channelID string) {
 // GetProviderChannel gets the channelID for the channel to the provider.
 func (k Keeper) GetProviderChannel(ctx sdk.Context) (string, bool) {
 	store := ctx.KVStore(k.storeKey)
-	channelIdBytes := store.Get(types.ProviderChannelKey())
-	if len(channelIdBytes) == 0 {
+	channelIDBytes := store.Get(types.ProviderChannelKey())
+	if len(channelIDBytes) == 0 {
 		return "", false
 	}
-	return string(channelIdBytes), true
+	return string(channelIDBytes), true
```

Comment:
Why wasn't this changed in the formatting and linting PRs from Notional that we merged a few weeks ago? Shouldn't be in this PR.